CONTRIBUTING.md

```diff
@@ -57,6 +57,22 @@ Note: `release/x.y` or `hotfix/x.y.z` will be merged back in `master` after a ne
 
 [git-flow]: http://nvie.com/posts/a-successful-git-branching-model/
 
+# Subtree'd library projects
+
+A number of library dependencies are managed as separate open source projects and are git-subtree'd into the WordPress Android app source tree. Use the following command to updated (pull latest) from their respective repos:
+
+        $ git subtree pull --prefix libs/library_name https://github.com/wordpress-mobile/WordPress-Library_Name-Android.git develop
```

Comment:
> It's a matter of preference anyway. More than happy to add the --squash if the consensus is towards that :)

My preference. If commits are already pushed in the subrepo' `develop` branch, that means they were already reviewed. So when we pull them here in the main repo, it makes the history cleaner and makes the PR easier to review.

> In my mind, there's going to be 2 groups of people: One with the people that has commit access and one with the contributors without commit access. The CONTRIBUTING.md file is mainly oriented towards the second group from which we expect contributions via Pull Requests and that's why I didn't include push info.

That was true at some point, but we have very few of them, and it serves as an internal documentation more than a guide to external contributors.